packages/docusaurus-plugin-content-docs/src/plugin-content-docs.d.ts

```diff
@@ -120,6 +120,12 @@ declare module '@docusaurus/plugin-content-docs' {
      * `undefined` to create a fully autogenerated sidebar.
      */
     sidebarPath?: string | false | undefined;
+    /**
+     * Path to the versioned docs content directory on the file system, relative to site
+     * directory. Running the tagging script `docusaurus docs:version X` will generate
+     * the files in this directory. The default is the site directory.
+     */
+    versionedDocsPath?: string | undefined;
```

Comment:
again not 100% agreeing with the comment: Docusaurus won't generate files in this directory but rather in 2 subdirectories of this directory (as far as I understand it)

Reply:
And if absolute paths are not supported, option validation should reject it. But absolute paths should probably be supported